types/hooks/screens/useSignedChannelListScreenHook.types.ts

```diff
@@ -1,11 +1,12 @@
 import {ChannelList} from '../../database/schema/ChannelList.types';
+import {ContactScreenPayload} from './useChatUtilsHook.types';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `ContactScreenPayload` from `useChatUtilsHook.types` is a good addition as it provides type safety for the payload passed to the `goToContactScreen` function.



<!-- This is an auto-generated comment by OSS CodeRabbit -->